tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -1736,18 +1791,19 @@ def test_build_fails_with_missing_metadata(self, runtime, use_container, layer_i
         self.assertEqual(command_result.process.returncode, 1)
         self.assertFalse(self.default_build_dir.joinpath(layer_identifier).exists())
 
-    @parameterized.expand([("python3.7", False, "LayerOne"), ("python3.7", "use_container", "LayerOne")])
-    def test_build_with_missing_buildarchitecture(self, runtime, use_container, layer_identifier):
+    @parameterized.expand([False, "use_container"])
+    def test_function_build_succeeds_with_referenced_layer(self, use_container):
```

Comment:
Looks like this test is changed to validate something else (it was building the layer before but now it is building the function)?

Reply:
This is because we elected not to warn on JUST a missing BuildArchitecture, which is what this test was doing before. I think the diff is misleading here because I wasn't changing a test, but rather replacing it entirely.